app/forms/openid_connect_token_form.rb

```diff
@@ -3,6 +3,8 @@ class OpenidConnectTokenForm
   include ActionView::Helpers::TranslationHelper
   include Rails.application.routes.url_helpers
 
+  ISSUED_AT_LEEWAY = 10.seconds
```

Comment:
I think rubocop is gonna get mad if we don't `.freeze` this?

Reply:
I think that's only for strings?

Reply:
oh good point